README.md

```diff
@@ -10,10 +10,23 @@ The Vault Operator deploys and initializes Vault on Kubernetes, and runs in high
 
 Deploy the charm:
 ```bash
-juju deploy vault-k8s -n 5 --trust
+juju deploy vault-k8s vault --channel edge --base jammy -n 5 --trust
```

Comment:
```suggestion
juju deploy vault-k8s --channel edge -n 5 --trust
```

Let's keep it simple, no need for specifying the base and changing the app name

Reply:
Done